### ISSUE TITLE
Return a list of fixes to original JSON

### ISSUE THREAD

**Issue by gloryknight** [josdejong/jsonrepair#79]
Thanks for the nice library.
`jsonrepair` returns a single value. To keep the API, I suggest to add a new function e.g. `jsonrepairer` which returns the fixed JSON and an array of fixes the library performed on the original JSON.
The list could be later inspected if somebody is interested in possible data corruptions.

**Comment by josdejong** [comment]
Yes, that would be really great to have the function also return a list with the applied fixes. Some fixes are more obvious than others, and it could be that certain situations can be interpreted in multiple ways. So maybe it is also possible to add information about that. Thanks for your suggestion 👍
